mptt/managers.py

```diff
@@ -672,6 +672,79 @@ def partial_rebuild(self, tree_id):
 
         self._rebuild_helper(pks[0], 1, tree_id)
 
+    @delegate_manager
+    def bulk_load(self, data, target=None, position='last-child'):
```

Comment:
Not convinced about the name - I think it's too close to `bulk_create` and it's not obvious that this is a special tree-related thing. If I was tab-completing in a shell I'd be unsure which one to pick.

Maybe something like:
* `build_tree_nodes`
* `load_tree_nodes`
* `tree_bulk_load`